website/docs/enterprise/pipelines/github-pr-promotions.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: Promoting apps
+sidebar_position: 3
+hide_title: true
+---
+
+import TierLabel from "../../_components/TierLabel";
+
+import CodeBlock from "@theme/CodeBlock";
+import BrowserOnly from "@docusaurus/BrowserOnly";
+
+# Promoting apps through pipeline environments <TierLabel tiers="enterprise" />
+
+Pipelines allow you to configure automatic promotions of applications through a consecutive set of environments, e.g. from dev to staging to production. The environments are defined in the `Pipeline` resource itself so that each pipeline governs a single application and all the environments it is deployed to.
```

Comment:
Perhaps we should explicitly call out "via pull request" if this will remain scoped to the pr strategy?